base/gcd_tests.cpp

```diff
@@ -0,0 +1,33 @@
+// LAF Base Library
+// Copyright (c) 2019 Igara Studio S.A.
+//
+// This file is released under the terms of the MIT license.
+// Read LICENSE.txt for more information.
+
+#include <gtest/gtest.h>
+
+#include "base/gcd.h"
+
+TEST(GreatCommonDivisor, RandomTests)
+{
+    EXPECT_EQ(base::gcd(98, 56), 14);
+    EXPECT_EQ(base::gcd(56, 98), 14);
+    EXPECT_EQ(base::gcd(9, 5), 1);
+    EXPECT_EQ(base::gcd(540,24), 12);
```

Comment:
Add missing whitespace between parameters (same for lines 18, 19)
```suggestion
  EXPECT_EQ(base::gcd(540, 24), 12);
```